vendor/engines/secure_rooms/app/controllers/secure_rooms_api/scans_controller.rb

```diff
@@ -22,9 +22,10 @@ def scan
 
     def load_user_and_reader
       @user = User.find_by!(card_number: params[:card_number])
+
       @card_reader = SecureRooms::CardReader.find_by!(
         card_reader_number: params[:reader_identifier],
-        control_device_number: params[:controller_identifier],
+        control_device_number: params[:controller_identifier].upcase,
```

Comment:
This actually won't really do much in MySQL because it's search is case-insensitive, but will affect Oracle. The test around this will pass on mysql with our without this `upcase`, but it should fail without it on Oracle.